kolibri/logger/test/test_api.py

```diff
@@ -387,6 +387,11 @@ def setUp(self):
         self.user2 = FacilityUserFactory.create(facility=self.facility)
         self.exam = Exam.objects.create(title="", channel_id="", question_count=0, collection=self.facility, creator=self.user2, active=True)
         self.examlog = ExamLog.objects.create(exam=self.exam, user=self.user1)
+        [ExamAttemptLog.objects.create(
+            item="d4623921a2ef5ddaa39048c0f7a6fe06", examlog=self.examlog, user=self.user1,
+            content_id=uuid.uuid4().hex, channel_id=uuid.uuid4().hex, start_timestamp=str(datetime.datetime.now()),
+            end_timestamp=str(datetime.datetime.now()), correct=0
```

Comment:
To make this somewhat more realistic, could you "jitter" these timestamps a bit? Maybe you could use the index in the `for x in in range(3)` to make these timestamps slightly different.